src/plugins/eqht-css/_base.scss

```diff
@@ -0,0 +1,17 @@
+.wb-eqht-grd{
+	align-items: stretch;
+	display: flex;
+	flex-wrap: wrap;
+	height: 100%;
+
+	>[class*="col-"]{
```

Comment:
We will add 2 additional selector,

```
>[class*="col-"], >[class*="col-"]>section, >section
```

Reply:
And then we will be able to remove the "hght-inhrt" repeated on every section in the first working example